server/src/metadata/tenant-migration/tenant-migration.service.ts

```diff
@@ -62,7 +62,7 @@ export class TenantMigrationService {
     workspaceId: string,
   ): Promise<TenantMigrationEntity[]> {
     return await this.tenantMigrationRepository.find({
-      order: { createdAt: 'ASC' },
+      order: { name: 'ASC' },
```

Comment:
Let's keep createdAt with the name, custom migrations (created by workspace members using the createOneObject endpoint) are not tied to a file so we still need to rely on the createdAt for that